src/Psalm/Codebase.php

```diff
@@ -1124,7 +1136,7 @@ public function getSymbolLocation(string $file_path, string $symbol): ?CodeLocat
 
             $file_contents = $this->getFileContents($file_path);
 
-            return new CodeLocation\Raw(
+            return new Raw(
```

Comment:
This is an example of where the readability does suffer. Could be renamed to `RawLocation` though.

Reply:
Yep, ``Raw`` is quite inexpressively.
The good thing is, it is used only 4 times. Anyway, ``DocblockTypeLocation`` and ``ParseErrorLocation`` are postfixed, so this should really be ``RawLocation``.

But I think this should go into an extra PR.

Reply:
> But I think this should go into an extra PR.

Of course.